src/operations.rs

```diff
@@ -684,44 +685,38 @@ fn remove_path_from_specific_file(path: PathBuf) -> Result<()> {
     Ok(())
 }
 
-pub fn find_shell_scripts_to_be_modified(add_case: bool) -> Result<Vec<PathBuf>> {
+pub fn find_shell_scripts_to_be_modified(add_case: bool) -> Vec<PathBuf> {
```

Comment:
I think it only can't return an error because my implementation is sloppy ;) The implementation right now has a bunch of `.unwrap()` statements in the function body, they should really be replaced with `?` and then the return type should be preserved, right?